zenodo/modules/snow/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2016 CERN.
```

Comment:
Unfortunately, this won't bring David Bowie back... You can change it to 2018 :) (look for other places as well)